### PR TITLE
feat : added cart functionality

### PR DESCRIPTION
**Pull request by Digambar1411**
Added cart functionality
functionality : Add to cart, remove from cart, price details
increment item in cart yet to be implement

files to check
pages/ecom-cart.js
pages/ecom-cart.css
contexts/cart-context
components/horizontalCard
components/productCard

### CODE REVIEW

File: src/frontend/pages/cart/ecom-cart.css [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    margin: 3rem auto;
+    gap: 5rem;
+    padding: 1rem;
+    flex-wrap: wrap;
+        
+}
+
+.empty-title{
+    margin: 0rem auto;
+    font-size: 1.5rem;
```
Comment (SanjayJatti) [inline src/frontend/pages/cart/ecom-cart.css:42]: liked that you used rem as unit

File: src/frontend/components/horizontalCard/horizontal-card.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
+import { useCart } from "../../contexts/cart-context";
+import "./horizontal-card.css";
+const laptopImage = require('../../assets/laptop.png');
+
+
+const HorizontalCard = (product)=>{
+  
+    const {removeFromCart}= useCart()
+    const incrementQuantity=(product)=>{
+        // {cart.map((item)=>item._id===product._id) ? {...item, quantity:product.quantity+1} :item
```
Comment (SanjayJatti) [inline src/frontend/components/horizontalCard/horizontal-card.js:10]: remove commented code

**Review by SanjayJatti — APPROVED** [review]
- Files are well structured
- Code is Clean

File: src/frontend/components/horizontalCard/horizontal-card.js (hunk 2 above)
Comment (Digambar1411) [reply src/frontend/components/horizontalCard/horizontal-card.js:10]: will do it next PR, this code is going to use for incrementing quantity

**Review by vijaytembugade — APPROVED** [review]
Looks good to me.
Good to merge

**Pull request merged**